src/test/java/com/easypost/UserTest.java

```diff
@@ -1,64 +1,141 @@
-/*
 package com.easypost;
 
 import com.easypost.model.*;
+
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Test;
-import org.junit.Rule;
-import org.junit.rules.ExpectedException;
 
-import java.text.ParseException;
 import java.util.HashMap;
 import java.util.Map;
-import java.util.List;
-import java.util.ArrayList;
-import java.util.Date;
-import java.text.DateFormat;
-import java.text.SimpleDateFormat;
-
-import java.lang.InterruptedException;
 
 import com.easypost.exception.EasyPostException;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 public class UserTest {
+    private static User user;
 
-  @BeforeClass
-  public static void setUp() {
-    EasyPost.apiKey = "PROD_KEY"; // PROD KEY REQUIRED
-  }
-
-  @Test
-  public void testRetrieveMe() throws EasyPostException {
-    User me = User.retrieveMe();
-
-    assertNotNull(me.getName());
-    assertNotNull(me.getEmail());
-  }
-
-  @Test
-  public void testCreateChild() throws EasyPostException {
-    User child = User.create();
-    assertNotNull(child.getName());
-    assertNotNull(child.getEmail());
-
-    List<ApiKey> childKeys = child.apiKeys();
-    assertNotNull(childKeys.get(0).getKey()); // one is test and one is prod
-    assertNotNull(childKeys.get(1).getKey()); // check getMode()
-  }
-
-  @Test
-  public void testEditUser() throws EasyPostException {
-    User me = User.retrieveMe();
-
-    Map<String, Object> params = new HashMap<String, Object>();
-    params.put("name", "New Name2736");
-    me.update(params);
-
-    assertEquals("User was not updated successfully", "New Name2736", me.getName());
-    System.out.println(me.prettyPrint());
-  }
-}
-*/
+    /**
+     * Setup the testing environment for this file.
+     *
+     * @throws EasyPostException when the request fails.
+     */
+    @BeforeClass
+    public static void setUp() throws EasyPostException{
+        EasyPost.apiKey = "EASYPOST_PROD_API_KEY";
+        user = User.retrieveMe();
+    }
+
+    /**
+     * Test creating a user.
```

Comment:
This should say `child user`
```suggestion
     * Test creating a child user.
```